.stickler.yml

```diff
@@ -1,2 +1,3 @@
 linters:
-  flake8: {  }
+  flake8:
+    max-line-length: 100
```

Comment:
cannot this be read from .flake file? or can we somehow use this linter in any editor like vim/emacs? Maybe we should create some CONTRIBUTE.md describing how to setup local envs.

Reply:
We will need to have probably both. I don't see any option to make it use `.flake8` file (which is introduced in #21 in fact).

Reply:
I'll add more information about installing the tool as part of #21. I just wanted it here to see how my changes will work. If you want, I can remove it from here and move it to #21.

Reply:
ok